core-api/src/main/java/com/optimizely/ab/config/DatafileProjectConfig.java

```diff
@@ -102,19 +103,20 @@ public class DatafileProjectConfig implements ProjectConfig {
     // v2 constructor
     public DatafileProjectConfig(String accountId, String projectId, String version, String revision, List<Group> groups,
                                  List<Experiment> experiments, List<Attribute> attributes, List<EventType> eventType,
-                                 List<Audience> audiences) {
-        this(accountId, projectId, version, revision, groups, experiments, attributes, eventType, audiences, false);
+                                 List<Audience> audiences, String region) {
```

Comment:
No region for V2 - remove this change
```suggestion
                                 List<Audience> audiences, String region) {
```